integrations/aws/CHANGELOG.md

```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 <!-- towncrier release notes start -->
 
+# Port_Ocean 0.2.16 (2024-07-16)
+
+### Improvements
+
+- Brute force global resource if does not have permissions in default region (#1)
```

Comment:
please use more appropriate explanation about what and why we chose to implement that